### ISSUE TITLE
Markdown: Not all `*` is converted into `_`

### ISSUE THREAD

**Issue by SimenB** [prettier/prettier#3170]
**Prettier 1.8.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqVARCACAnhAV2wHcBDWbGHAJzgEcC4BnGbU7VAMzlJgNtTYI1bLQAOw1u1QAjAgHNUAfnQgANCAhiYAS2hNkoUi2SdSAGyZwNM6qTABrODADKY+zqjzkMaow0AJhBgphZWGgBWTAAeAEJ2js4upAC2cAAynnChltaaBDBiBQBMOeEg7tRW1MggKaTUDkHEUOoV1J4wAOo6ATAAFsgAHAAMGlYpOj5+eUye8uZwAIoEEPBleTCkMj19g0jFGr6kOubzAMIQKfW1UNDZGgRWACrbBkhmuQC+X0A)
```sh
--parser markdown
```

**Input:**
```markdown
**Do you want to request a *feature* or report a *bug*?**
```

**Output:**
```markdown
**Do you want to request a _feature_ or report a *bug*?**

```

**Expected behavior:**
Both `*feature*` and `*bug*` should be converted

**Comment by azz** [comment]
Minimal repro:
**Prettier 1.8.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqARgVwOaoPwgA0IEADjAJbQDOyoAhtTMjAE6ZzHqv1gDWcGAGVSvClGwt2nEABMIYZADN6AG2oyAVtQAeAIR79BQ+gFs4AGXFxlajcQiYYpJwCZb6maNYbWyEKb0rHzyAO5QRCCkrOIwAOoUsjAAFsgAHAAMxBqmFFIc2eLYqnAAipgQ8B72IDD06AlJqUiuxGz0FKpFAMIQpoH+UNA2xJgaACr1tEgqngC+c0A)
```sh
--parser markdown
```

**Input:**
```markdown
*bug*?
```

**Output:**
```markdown
*bug*?

```

**Comment by SimenB** [comment]
Related, should `**` be converted to `__`?

**Comment by azz** [comment]
The other way around is expected.

**Prettier 1.8.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqVAjArgc3QHSkIH1jsdSQAaECABxgEtoBnZUAQxZmRgCcscGhj4cwAazgwAynTGMoOXgKEgAJhDDIAZhwA2LVQCsWADwBCoiVOkcAtnAAyCuDv2GaELDDreATG4GqnJ8hnzIIHYcfOIaAO5Q1CB0fAowAOqMajAAFsgAHAAMNIZ2jMqCJQo4enAAilgQ8IEeIDAcGJnZeUh+NPwcjHrVAMIQdlERUNCuNFiGACodbEi6QQC+60A)
```sh
--parser markdown
```

**Input:**
```markdown
**bug**

__bug__
```

**Output:**
```markdown
**bug**

**bug**

```

**Comment by SimenB** [comment]
Why isn't prettier consistent with what syntax it uses for emphasis?

**Comment by azz** [comment]
It's based on most common usage.  https://github.com/prettier/prettier/pull/2943#issuecomment-333366584

**Comment by azz** [comment]
@ikatyang could you take a look at this one, too? I've got to do something for the next ~1h but I'd like to release [1.8.1](https://github.com/prettier/prettier/milestone/4) after that.

**Comment by azz** [comment]
This has been fixed in `1.8.1`.
